java/code/src/com/redhat/rhn/common/hibernate/AbstractConnectionManager.java

```diff
@@ -206,7 +206,7 @@ protected void createSessionFactory() {
 
             sessionFactory = config.buildSessionFactory();
         }
-        catch (HibernateException e) {
+        catch (Exception e) {
```

Comment:
Don't you have a more specific exception to catch? Sonarcloud usually considers those generic catches as bad.

Reply:
right, better use by now HibernateException, then I can add new exception when they will happen again